docs/setup/aks/setup-aks-appinsights-grafana.md

```diff
@@ -191,17 +191,19 @@ kubectl --namespace default port-forward <REPLACE-WITH-GRAFANA-POD-NAME> 3000
 ```
 Now, browse to http://127.0.0.1:3000/ and log in using the admin username and password you provided in the `values.yaml` file.
 
+
 Verify that `Azure Monitor` exists in the plugins page.
 
 Configure a data source using the [plugin support page](https://github.com/grafana/azure-monitor-datasource#configure-the-data-source).
 
-Finally, see the provided sample Grafana JSON file (**[grafana-sample-dashboard.json](../../assets/grafana-sample-dashboard.json)**), which you can import from the Grafana dashboard: from the menu (plus icon - `+`) select **Create > Import** and upload the sample JSON file.
+You can import a sample Grafana JSON file from the Grafana dashboard: from the menu (plus icon - `+`) select **Create > Import** and upload the sample JSON file.
+
```

Comment:
same - extra new line

Reply:
leftover from my attempts to get the file to download. deleted